### DISCUSSION TITLE
Could it be that account_balances is not being calculated correctly?

### DISCUSSION THREAD

**Discussion by lvitti** [firefly-iii/firefly-iii#9417] [Firefly III] [answered]
I've been using a direct connection to the database for a while to generate custom reports. 
However, I just realized that the `account_balances` table is no longer being updated, and it’s one of the tables I'm using in my reports.

Has anything changed? Is there something I need to do to obtain the account balances?

**Comment by JC5** [comment, marked as answer]
Hey, thanks for opening a discussion. 

The account balances is very experimental and not yet a finished feature. It's not documented. 

You can set USE_RUNNING_BALANCE=true and if you restart the container it should fill the table again. But again, experimental feature :)

**Reply by lvitti** [reply]
Understood !

I activated that setting and restarted, but it still has incorrect data, and new transactions aren’t updating those values.

What do you recommend as the best way to obtain the current account balance?

Thanks!

**Reply by JC5** [reply]
Something like `select sum(amount) from transactions where deleted_at is not null group by transaction_currency_id where account_id = x`

**Reply by lvitti** [reply]
Thanks !
